src/pages/RepoPage/FailedTestsTab/FailedTestsPage/MetricsSection/MetricsSection.tsx

```diff
@@ -83,6 +86,9 @@ const TotalTestsRunTimeCard = ({
           <PercentBadge value={totalDurationPercentChange} />
         ) : null}
       </MetricCard.Content>
+      <MetricCard.Description>
+        The cumulative CI time spent running tests over the last {intervalCopy}
```

Comment:
period at the end here to be consistent with the other descriptions?

Reply:
Nice catch! Will fix